rllib/policy/sample_batch.py

```diff
@@ -280,6 +248,13 @@ def __init__(self, *args, **kwargs):
         if self._is_training is None:
             self._is_training = self.pop("is_training", False)
 
+        for k, v in self.items():
+            # TODO: Drop support for lists and Numbers as values.
+            # Convert lists of int|float into numpy arrays make sure all data
+            # has same length.
+            if isinstance(v, (Number, list)):
+                self[k] = np.array(v)
+
```

Comment:
This was previously part of our counting code. Moving it does not yield a performance improvement but we should still move it.